src/components/Cards/Cards.tsx

```diff
@@ -1,11 +1,59 @@
 import React, { FC } from "react";
-import { Link } from "react-router-dom";
-import { product } from "@constants";
+import { Rate } from "antd";
+import { FoodApi } from "services/FoodService";
+import { HeartOutlined } from "@ant-design/icons";
+import s from "./Cards.module.scss";
 
 const Cards: FC = () => {
+  const { data, isLoading, error } = FoodApi.useFetchAllFreshFoodQuery();
+  console.log(data);
   return (
-    <div>
-      <Link to={product + 1234}>Cards</Link>
+    <div className={s.cards}>
+      <div className={s.cards__item}>
+        {data?.map((el) => (
+          <div className={s.card}>
+            <div className={s.card__img}>
+              <img src={`https://spoonacular.com/cdn/ingredients_500x500/${el.image}`} alt="" />
+            </div>
+            <div className={s.card__body}>
+              <h3 className={s.card__title}>{el.name}</h3>
+              <p className={s.card__simpleText}>{el.description.slice(0, 20).trim().concat("...")}</p>
+              <Rate disabled defaultValue={el.popularity} className={s.card__stars} />
+              <ul className={s.card__info}>
+                <li className={s.card_lightText}>
+                  Fresheness <span className={s.card__span}>...</span> {el.fresheness}
+                </li>
```

Comment:
Think, it isn't the best solution

Reply:
In variables.scss

Reply:
I thought about this for a long time, and I didn't understand how to make a couple of spaces together, so this method was the last option (

Reply:
I can use &nbsp;

Reply:
But if i will use & nbsp; , I have to duplicate this operator for every space , i can't use "repeat" with it .Also I can't use {' '} because vscode removes them immediately after saving, so I have no other options (

Reply:
Changes added